examples/inverse/plot_psf_ctf_vertices_lcmv.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+"""
+Compute cross-talk functions for LCMV beamformers
+==============================================================
+
+Visualise cross-talk functions at one vertex for different LCMV beamformers.
```

Comment:
can you explain a bit more what this shows and what to look at. Does it tell anything
about the different beamformers?

Reply:
I now write that different covariance matrices produce different cross-talk functions. I could come up with a more meaningful example (e.g. comparison LCMV with MNE), but maybe this would be better for a tutorial?